docs/get-started/lotus/installation.md

```diff
@@ -187,6 +187,10 @@ One example is that logs are redirected to files in `/var/log/lotus` by default
 
 These instructions are specific to macOS. If you are installing Lotus on a Linux distribution, head over to the [Linux section](#linux).
 
+:::warning
+Macs with the M1 chip are not currently supported.
+:::
```

Comment:
@dignifiedquire I believe that @johnnymatthews is challenging you :D

Reply:
Haha :smile:

Reply:
work is happening here: https://protocollabs.slack.com/archives/CRS9PSD53/p1618005466279300